lib/tasks/import_data_to_bigquery.rake

```diff
@@ -1,7 +1,32 @@
 DEFAULT_BATCH_SIZE = 200
 SLEEP_TIME = 2
+IRREGULAR_TABLE_NAMES = {
+  provider_relationship_permissions: 'ProviderRelationshipPermissions',
+  provider_user_notifications: 'ProviderUserNotificationPreferences',
+  provider_users_providers: 'ProviderPermissions',
+  references: 'ApplicationReference',
+}.freeze
```

Comment:
:+1: we can decide how to centralise this later

Reply:
Yes. There is still a bit of refactoring to do, just need to test this in QA as my local data are not optimal and I was also getting some SSL and some unreachable host errors that I think may have been caused by my connection being a bit flaky.